### PR TITLE
Add info of authors of each project in about section

### PR DESCRIPTION
**Pull request by santisoler**
Now `conf.py` generates a set of Markdown files containing html snippets with information of the authors of each project as Bootstrap cards.
The about section shows this info under Boostrap pills, one pill for each project.
Add the authors files that are dynamically created to `.gitignore`.
Tell livereload to ignore these authors files.
Use a dummy placeholder for any missing avatar picture.

### CODE REVIEW

File: conf.py [hunk 1]
```diff
@@ -1,10 +1,112 @@
...
+import re
 import datetime
 import subprocess
+from requests import get
+from pathlib import Path
+
+
+AUTHORS_BASE_URL = "https://raw.githubusercontent.com/fatiando/{}/{}/AUTHORS.md"
+AUTHOR_HTML_CARD = """
+<div class="col-6 col-md-3 col-sm-4">
```
Comment (leouieda) [inline conf.py:13]:
```suggestion
<div class="col-4 col-sm-3 col-md-2 gx-2">
```

Reduce the column size so we can fit more cards per line. This still looks good but doesn't take up too much vertical space.

File: conf.py [hunk 2]
```diff
@@ -1,10 +1,112 @@
...
+  <div class="card">
+    <img
+        class="card-img-top"
+        src="{avatar_url}"
+        alt="Profile picture of {full_name}"
+    >
+    <div class="card-body">
+      <h4 class="card-title fs-5">
```
Comment (leouieda) [inline conf.py:21]:
```suggestion
      <h4 class="card-title fs-6">
```

Again, smaller size to better fit the smaller columns.

**Review by leouieda — COMMENTED** [review]
Looks really good @santisoler! I made a small suggestion to make the cards a bit smaller so they don't dominate the page that much.

I also noticed that the page load time has increased quite a bit because of all the images. In the future, we should consider downloading the pictures and reducing their resolution (convert to jpg also) and store them to help with that.

**Comment by santisoler** [comment]
Thanks for the suggestions!

I experienced the same thing with the loading time. The placeholder image loads very quickly while you can easily witness how the other ones are still being loaded. I agree we should download them and convert them to jpg files to spare a few seconds and bandwidth while loading the website.

If it's ok with you, we could merge this PR and open a new one for caching the images.

**Comment by leouieda** [comment]
👍🏽 yep that was my idea

**Pull request merged**